docs/backup_workstations.rst

```diff
@@ -94,43 +94,35 @@ mounted and ready to access.
 
 |Backup and TailsData Mounted|
 
-Open a Nautilus window with administrative privileges by first going to
+Open a terminal with administrative privileges by going to
 **Applications** ▸ **System Tools** ▸ **Root Terminal**.
 
 |Root Terminal|
 
-Type ``nautilus`` at the terminal prompt and hit enter. You'll need to type in
-your administrative passphrase you set at the Tails login screen.
-
-|Start Nautilus|
-
-.. note:: When you run ``nautilus``, you may run into an error where Nautilus
- complains that it can't create a required folder. If that happens, just
- click OK and continue normally.
-
+.. note:: 
  If you can't find the "Root Terminal" window, it might be because an
  administrator passphrase wasn't set when you logged in to Tails. If
  that's the case, you'll need to restart Tails and set one at the login screen.
 
-.. warning:: Make sure you use keep the `Root Terminal` window open while
- you perform the backups. Otherwise, the `Nautilus` window will close.
+Next, create a directory on the Backup USB for the device to be backed up - the
+command below creates a directory named ``admin-backup``:
```

Comment:
Possible nit: does Tails name all devices `Media` by default, or should we maybe call this `MyUsb` for extra clarity?

Reply:
Tails doesn't name all devices `Media` by default, but _will_ mount all USB drives inside `/media/` by default. As the docs stand, the path `/media/amnesia/Backup` matches the earlier instruction to name the drive "Backup", including screenshots indicating the same name.